utils/zerovec/src/ule/multi.rs

```diff
@@ -4,21 +4,20 @@
 
 use super::*;
 use crate::varzerovec::lengthless::VarZeroLengthlessSlice;
-use crate::varzerovec::Index32;
 use crate::vecs::VarZeroVecFormat;
 use core::{fmt, mem};
 
 /// This type is used by the custom derive to represent multiple [`VarULE`]
 /// fields packed into a single end-of-struct field. It is not recommended
-/// to use this type directly.
+/// to use this type directly, use [`Tuple2VarULE`](crate::ule::tuplevar::Tuple2VarULE) etc instead.
 ///
 /// Logically, consider it to be `(V1, V2, V3, ..)`
 /// where `V1` etc are potentially different [`VarULE`] types.
 ///
-/// Internally, it is represented by a VarZeroSlice.
+/// Internally, it is represented by a VarZeroSlice without the length part.
 #[derive(PartialEq, Eq)]
 #[repr(transparent)]
-pub struct MultiFieldsULE<const LEN: usize, Format: VarZeroVecFormat = Index32>(
+pub struct MultiFieldsULE<const LEN: usize, Format: VarZeroVecFormat>(
```

Comment:
Suggestion: Make sure all the safety comments say that the `Format` needs to be the same between validation and reading fields. It seems like a footgun that the MultiFieldsULE could be potentially deserialized with one format and read with another format. Making the format be explicit is probably a positive change.

Reply:
I updated the comments to explicitly mention the format where it talks about VZLS. I also made VZLS require the format.